src/attpc_engine/detector/transporter.py

```diff
@@ -324,15 +324,16 @@ def transport_track(
         the Nth time step.
     electrons: np.ndarray
         1xN array of electrons created each time step (point) of the trajectory.
+    points: numba.typed.Dict
+        A dictionary mapping a unique pad,tb key to the number of electrons.
 
     Returns
     -------
-    numpy.ndarray
-        An Nx3 array representing the simplified point cloud
+    points: numba.typed.Dict[int, int]
```

Comment:
Same as above, I don't think we should return the dictionary. We fill it in each function, so no need to return a copy (its the same dictionary).